website/docs/reference/global-configs.md

```diff
@@ -3,49 +3,61 @@ title: "Global Configs"
 id: "global-configs"
 ---
 
-dbt supports several global runtime configs. Starting in v1.0, all these configs can be set in three places:
-- `config:` block in `profiles.yml`, sometimes called "user config"
-- environment variables, prefixed with `DBT_`
-- CLI flags, which immediately follow `dbt` and precede your subcommand
+### About Global Configs
 
-Notes:
-- The precedence order is always (1) CLI flag > (2) env var > (3) profile config
-- All boolean configs can be turned on via a CLI flag named `--this-config`, and turned off with a CLI flag named `--no-this-config`
-- Use the profile config to set defaults for all projects running on your local machine
+Global runtime configurations (global configs) enable you to set default configuration values for all projects that you run locally on your computer. dbt supports several global configs.
```

Comment:
I got this info [here](https://deploy-preview-913--docs-getdbt-com.netlify.app/reference/profiles.yml#user-config). Do you have anything we can say to enhance this?

Reply:
Maybe worth saying a bit about what the purpose of these configs is. Something like:
> Global configs control things like the visual output of logs, the manner in which dbt parses your project, and what to do when dbt finds a version mismatch or a failing model.
> 
> Global configs enable you to fine-tune _how_ dbt runs projects on your machine—whether your personal laptop, an orchestration tool running remotely, or (in some cases) dbt Cloud. They differ from [project configs](reference/dbt_project.yml) and [resource configs](reference/configs-and-properties), which tell dbt _what_ to run.
>
> These configs are "global" because they are available for all dbt commands, and because they apply across all projects run on the same machine.